src/app/games/game/game.component.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Game} from '../../store/model';
+
+@Component({
+  selector: 'app-game',
+  templateUrl: './game.component.html',
+  styleUrls: ['./game.component.scss']
+})
+export class GameComponent implements OnInit {
+
+  @Input() public game: Game;
+  private imgSrc: string;
+
+  ngOnInit(): void {
+    this.imgSrc = 'https://images.igdb.com/igdb/image/upload/t_cover_big/' + this.game.imageId + '.jpg';
```

Comment:
[suggestion] Use template strings, i.e.
```
`https://images.igdb.com/igdb/image/upload/t_cover_big/${this.game.imageId}.jpg`
```

Also: the image path could be moved to a constant.